package.json

```diff
@@ -29,5 +30,30 @@
     "typescript": "^3.4.2",
     "webpack": "^4.29.6",
     "webpack-cli": "^3.3.0"
+  },
+  "devDependencies": {
+    "@matthid/source-map-visualization": "^0.5.0",
+    "nyc": "^14.1.1",
+    "source-map-support": "^0.5.12"
+  },
```

Comment:
Not very important, but in principle all the dependencies here are `devDependencies` because the root package.json is not used to publish any package. That's why I'm not making a distinction.

Reply:
Ah ok I wondered about this and left it as is to trigger this comment I guess ;)